src/transformers/models/paligemma/processing_paligemma.py

```diff
@@ -124,25 +152,8 @@ def __call__(
         self,
         text: Union[TextInput, PreTokenizedInput, List[TextInput], List[PreTokenizedInput]] = None,
         images: ImageInput = None,
```

Comment:
These two inputs should be reversed and support for backward compatibility should be added. This should be similar to what is needed for Fuyu:
https://github.com/huggingface/transformers/blob/aa3bc0b4d5c98a40fc211994697d1893976f8bf9/src/transformers/models/fuyu/processing_fuyu.py#L522-L532